common/scala/src/main/scala/whisk/core/entity/Subject.scala

```diff
@@ -17,15 +17,19 @@
 package whisk.core.entity
 
 import scala.util.Try
+
+import spray.json.JsString
 import spray.json.JsValue
 import spray.json.RootJsonFormat
-import spray.json.JsString
 import spray.json.deserializationError
+import whisk.core.entitlement.Privilege.Privilege
+import whisk.core.entitlement.Privilege
 
 protected[core] class Subject private (private val subject: String) extends AnyVal {
     protected[core] def apply() = subject
     protected[core] def namespace = EntityPath(subject)
     protected[entity] def toJson = JsString(subject)
+    protected[core] def toIdentity = Identity(this, EntityName(subject), AuthKey(), Privilege.ALL)
```

Comment:
Not sure about this, as it creates a "dummy" `Identity` (for testing?) which might be misleading in usual coding.